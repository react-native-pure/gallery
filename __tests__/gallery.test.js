'use strict';

import React from 'react'
import {render} from 'react-native-testing-library'
import App from '../Example/App'

describe('TreeModal', () => {
    it('renders correctly', () => {
        const instance = render(
            <App/>
        );
        expect(instance.toJSON()).toMatchSnapshot();
    });
});
