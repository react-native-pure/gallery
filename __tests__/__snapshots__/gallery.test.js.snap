// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TreeModal renders correctly 1`] = `<View />`;
